openslides/urls.py

```diff
@@ -31,7 +31,7 @@
     (r'^agenda/', include('openslides.agenda.urls')),
     (r'^motion/', include('openslides.motion.urls')),
     (r'^assignment/', include('openslides.assignment.urls')),
-    (r'^user/', include('openslides.users.urls')),
+    (r'^users/', include('openslides.users.urls')),
```

Comment:
This line is deprecated. The change you might want to do is in line 16 of this file.